Sources/StreamVideo/Errors/Errors.swift

```diff
@@ -4,8 +4,6 @@
 
 import Foundation
 
-extension APIError: Error {}
```

Comment:
We can't remove this unless we change the code generation logic.

Reply:
Very good point! I have forgotten this is generated